### DISCUSSION TITLE
"View as" in StaffCP

### DISCUSSION THREAD

**Discussion by StormyIceLeopard** [NamelessMC/Nameless#2157] [Ideas]
Let's say we want to see how the website looks/acts for another group, it would be nice if we could easily make a temp dummy account to use or a "view as" button to temporarily change our group so we can see what the target group sees.

This would be great to help ensure we are setting up correct permissions.

**Comment by tadhgboyle** [comment]
I really like this, but I am not sure how to implement it due to how permissions are checked. Will look into it
